jupyterhub/templates/image-puller/_daemonset-helper.yaml

```diff
@@ -110,9 +97,6 @@ spec:
         {{- range $k, $container := .Values.singleuser.extraContainers }}
         - name: image-pull-singleuser-extra-container-{{ $k }}
           image: {{ $container.image }}
-          {{- with $container.imagePullPolicy }}
-          imagePullPolicy: {{ . }}
-          {{- end }}
```

Comment:
I figured that it doesn't make sense for the prePuller to have a imagePullPolicy configured like the singleuser has it configured, so I removed these from here. I'm not 100% on this, but it also reduced the complexity and I could not see it turning out to be especially bad.